apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -148,11 +146,8 @@ else
   xctest_binary="$platform_developer_dir/Library/Xcode/Agents/xctest"
   test_file=$(file "$test_tmp_dir/$test_bundle_name.xctest/$test_bundle_name")
   if [[ "$intel_simulator_hack" == true ]]; then
-    sliced_xctest_binary=/tmp/xctest_intel
-    if [[ ! -x "$sliced_xctest_binary" ]]; then
-      lipo -thin x86_64 -output "$sliced_xctest_binary" "$xctest_binary"
-    fi
-
+    sliced_xctest_binary="$test_tmp_dir/xctest_intel_bin"
```

Comment:
Is `test_tmp_dir` per test? How expensive is the slicing?

Reply:
correct.

```
% time lipo -thin x86_64 -output /tmp/slice $(xcode-select -p)/Platforms/iPhoneSimulator.platform/Developer/Library/Xcode/Agents/xctest
lipo -thin x86_64 -output /tmp/slice   0.00s user 0.00s system 53% cpu 0.009 total
```